sdk/slot-history/src/lib.rs

```diff
@@ -0,0 +1,206 @@
+//! A type to hold data for the [`SlotHistory` sysvar][sv].
+//!
+//! [sv]: https://docs.solanalabs.com/runtime/sysvars#slothistory
+//!
+//! The sysvar ID is declared in [`sysvar`].
+//!
+//! [`sysvar`]: crate::sysvar
```

Comment:
This will get done in future work, once all the sysvar types are extracted

Reply:
Let's update the doc link?

Reply:
Sure, done